community/cypher/cypher/src/test/scala/org/neo4j/cypher/RootPlanAcceptanceTest.scala

```diff
@@ -146,7 +146,8 @@ class RootPlanAcceptanceTest extends ExecutionEngineFunSuite {
   }
 
   test("EstimatedRows should be properly formatted") {
-    given("match n return n").planDescripton.getArguments.get("EstimatedRows") should equal(0)
+    given("match n return n").planDescripton.getArguments.get("EstimatedOperatorCardinality") should equal(0)
+    given("match n return n").planDescripton.getArguments.get("EstimatedProducedRows") should equal(0)
```

Comment:
Sure I am open to suggestion for better names for the columns.

About docs, you are absolutely right, but not now since this is a preview which we are not gonna merge. Let's write docs for the final version of this feature.